html/recipes.js

```diff
@@ -1118,6 +1122,122 @@ export const recipes = {
 		note: 'Restores 30 sanity over 1 minute',
 		mode: 'together'
 	},
+	koalefig_trunk: {
+		name: 'Fig-Stuffed Trunk',
+		test: (cooker, names, tags) => {
+			return (names.trunk_summer || names.trunk_cooked || names.trunk_winter) && (names.fig || names.fig_cooked);
+		},
+		requirements: [NAME('trunk_summer'), NAME('fig')],
+		priority: 40,
+		foodtype: 'meat',
+		health: healing_huge,
+		hunger: calories_large + calories_medsmall,
+		sanity: 0,
+		perish: perish_med,
+		cooktime: 2,
+		mode: 'together'
+	},
+	figatoni: {
+		name: 'Figatoni',
+		test: (cooker, names, tags) => {
+			return (names.fig || names.fig_cooked) && tags.veggie && tags.veggie >= 2 && !tags.meat;
+		},
+		requirements: [NAME('fig'), TAG('veggie', COMPARE('>=', 2)), NOT(TAG('meat'))],
+		priority: 30,
+		foodtype: 'veggie',
+		health: healing_medlarge,
+		hunger: calories_large + calories_medsmall,
+		sanity: sanity_med,
+		perish: perish_fast,
+		cooktime: 2,
+		mode: 'together'
+	},
+	figkabab: {
+		name: 'Figkabab',
+		test: (cooker, names, tags) => {
+			return (names.fig || names.fig_cooked) && names.twigs && tags.meat && tags.meat >= 1 && (!tags.monster || tags.monster <= 1);
+		},
+		requirements: [NAME('fig'), SPECIFIC('twigs'), TAG('meat', COMPARE('>=', 1)), OR(NOT(TAG('monster')), TAG('monster', COMPARE('<=', 1)))],
+		priority: 30,
+		foodtype: 'meat',
+		health: healing_med,
+		hunger: calories_med,
+		sanity: sanity_med,
+		perish: perish_slow,
+		temperature: hot_food_bonus_temp,
+		temperatureduration: food_temp_long,
+		note: 
```

Comment:
```suggestion
```

Reply:
Oh right, I probably wanted to add a note that it gives temperature and either forgot or decided against it because the guide does add a note when the temperature stuff is given. However, when compared to the wiki (and I checked in game that the wiki makes more sense), the automatic note doesn't make much sense, so that could use a revamp. It says something like "provides 40 heat for 5 seconds" (or -40 and different time, depends on the dish), when what seems to happen is that dishes raise or lower the temperature by one degree per second, for a certain amount of time.